pkg/report/title_to_type_test.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2025 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package report
+
+import "testing"
+
+func TestTitleToTypeDefinitions(t *testing.T) {
+	knownPrefixes := make(map[string]bool)
+	for _, def := range titleToType {
+		if len(def.includePrefixes) == 0 {
+			t.Errorf("title definition can't be empty")
+		}
+		for _, prefix := range def.includePrefixes {
+			if prefix == "" {
+				t.Errorf("title prefix can't be empty")
+			}
+			if knownPrefixes[prefix] {
+				t.Errorf("duplicate title prefix: %q", prefix)
+			}
+			if wasMatched, byPrefix := hasPrefix(knownPrefixes, prefix); wasMatched {
+				t.Errorf("%s was matched by %s", prefix, byPrefix)
+			}
+			knownPrefixes[prefix] = true
+		}
+	}
+}
+
+// hasPrefix checks whether prefixes contains any of the s prefixes.
+func hasPrefix(prefixes map[string]bool, s string) (bool, string) {
+	for prefix := range prefixes {
+		for i := 1; i < len(s); i++ {
```

Comment:
`strings.HasPrefix(s, prefix)`? Or am I missing something?

Reply:
It doesn't return the string that has this prefix. I need it to make the error message more actionable.
But I didn't know about its existence. Thanks.